<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Móveis</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>85</width>
       <height>65</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="rb_mesa">
        <property name="text">
         <string>Mesa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_cadeira">
        <property name="text">
         <string>Cadeira</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_escrivaninha">
        <property name="text">
         <string>Escrivaninha</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Verificar</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>84</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="rb_aviao">
       <property name="text">
        <string>Avião</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_carro">
       <property name="text">
        <string>Carro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_navio">
       <property name="text">
        <string>Navio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>84</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rb_vermelho">
       <property name="text">
        <string>vermelho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_verde">
       <property name="text">
        <string>Verde </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_azul">
       <property name="text">
        <string>Azul</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>336</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
